cg/cli/store.py

```diff
@@ -25,26 +32,23 @@ def store():
 def fastq_cmd(context, case_id, dry_run):
     """Store fastq files for a case in Housekeeper"""
     LOG.info("Running store fastq")
-    cg_store = Store(context.obj.get("database"))
 
-    hk_api = hk.HousekeeperAPI(context.obj)
-    scout_api = scoutapi.ScoutAPI(context.obj)
-    crunchy_api = crunchy.CrunchyAPI(context.obj)
+    compress_api = context.obj["compress"]
+    cg_store = context.obj["db"]
 
-    compress_api = CompressAPI(hk_api=hk_api, crunchy_api=crunchy_api, scout_api=scout_api)
     update_compress_api(compress_api, dry_run=dry_run)
 
     samples = get_fastq_individuals(cg_store, case_id)
 
-    stored_inds = 0
+    stored_individuals = 0
     try:
         for sample_id in samples:
             was_added = compress_api.add_decompressed_fastq(sample_id)
             if was_added is False:
                 LOG.info("skipping individual %s", sample_id)
                 continue
-            stored_inds += 1
+            stored_individuals += 1
     except CaseNotFoundError:
         return
```

Comment:
Do we know for sure if this gives rise to some logging? Maybe CaseNotFoundError could be thrown with a message, which would be logged here, instead of somewhere in compress_api? I might be wrong though, just something to consider

Reply:
It is logged in the function that is run. This is also tested if you check the test